[Actions][Doc] Clean up Actions README (#91789)
elastic/kibana: docs/user/alerting/action-types/jira.asciidoc

Jira actions have the following configuration properties:

Subaction::        The subaction to perform. Either `pushToService`, `getIncident`, `issueTypes`, `fieldsByIssueType`, `issues`, `issue`, or `getFields`.
Subaction Params:: The parameters of the subaction.

==== `pushToService` subaction configuration

Incident:: The Jira incident which has the following properties:
* `summary` - The title of the issue.

> * `description` - The description of the issue.

Reviewer: ```suggestion
* `description` - A description of the issue.
```

> * `externalId` - The id of the issue in Jira. If present the issue will be updated. Otherwise a new issue will be created.

Reviewer: ```suggestion
* `externalId` - The ID of the issue in Jira. If present, the issue is updated. Otherwise, a new issue is created.
```

> * `issueType` - The id of the issue type in Jira.

Reviewer: ```suggestion
* `issueType` - The ID of the issue type in Jira.
```

> * `priority` - The name of the priority in Jira. Example: `Medium`.

Reviewer: ```suggestion
* `priority` - The priority level in Jira. Example: `Medium`.
```

* `labels` - An array of labels. Labels cannot contain spaces.

> * `parent` - The parent issue id or key. Only for Sub-task issue types.

Reviewer: ```suggestion
* `parent` - The parent issue ID or key. Only for subtask issue types.
```

> Comments:: The comments of the case. A comment is of the form `{ commentId: string, version: string, comment: string }`.

Reviewer: ```suggestion
Comments:: A comment in the form of `{ commentId: string, version: string, comment: string }`.
```

==== `getIncident` subaction configuration

> External ID:: The id of the issue in Jira.

Reviewer: ```suggestion
External ID:: The ID of the issue in Jira.
```

==== `issueTypes` subaction configuration

> No parameters for `issueTypes` subaction. Provide an empty object `{}`.

Reviewer: ```suggestion
The `issueTypes` subaction has no parameters. Provide an empty object `{}`.
```

==== `fieldsByIssueType` subaction configuration

> ID:: The id of the issue in Jira.

Reviewer: ```suggestion
ID:: The ID of the issue in Jira.
```

==== `issues` subaction configuration

Title:: The title to search for.

==== `issue` subaction configuration

> ID:: The id of the issue in Jira.

Reviewer: ```suggestion
ID:: The ID of the issue in Jira.
```

==== `getFields` subaction configuration

> No parameters for `getFields` subaction. Provide an empty object `{}`.

Reviewer: ```suggestion
The `getFields` subaction has no parameters. Provide an empty object `{}`.
```

Reviewer: This section is a little hard to read as formatted.  Try putting the parameter and description on the same line.

Author: Which section exactly? The subaction configuration parameters?
